src/components/virtual-gamepad-controls.css

```diff
@@ -19,10 +23,14 @@
   height: 0;
   left: 0;
   right: 0;
-  bottom: 10vh;
+  bottom: 15vh;
   display: flex;
   align-items: center;
   justify-content: space-around;
+
+  @media (orientation: portrait) {
+    display: none;
+  }
```

Comment:
I would much prefer if we kept the joysticks in portrait mode, if we could.

Reply:
I can see how it feels, but I figured it would be weird given the chatbox collides with it. We'd at least have to hide the preview sticks.